Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7.3-slim
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV AIRFLOW_HOME=/src/airflow
+
+WORKDIR /src/app
+
+COPY ./requirements ./requirements
+
+RUN pip install -U pip \
+    && set -ex \
+    && apt-get update -yq \
+    && apt-get install -yq --no-install-recommends build-essential \
+    && pip install --no-cache-dir -r ./requirements/base.txt \
+    && apt-get remove --purge --autoremove -yq build-essential \
+    && rm -rf ~/.cache/* \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/*
+
+ARG env
+RUN pip install --no-cache-dir -r ./requirements/${env}.txt \
+    && rm -rf /tmp/*
+
+EXPOSE 8080
```

Comment:
reason for it not being `80`? The host can always forward 8080 to the container's 80, but maybe 80 is being reserved for something else.

Reply:
I was just making airflow available to the host browser. I have also done the same for localstack on port 8081. I think it keeps the urls consistent by having one at 8080 and the other at 8081.

Reply:
:+1: for consistency between host and container port in local development